src/test/ui/try-trait/bad-interconversion.stderr

```diff
@@ -16,7 +16,7 @@ LL |     Ok(Err(123_i32)?)
              <f64 as From<i16>>
              <f64 as From<i32>>
              <f64 as From<i8>>
-           and 67 others
+           and 68 others
```

Comment:
Oh yeah.  Weird implications of impls on stable types is why I didn't have these in the `pub(crate)` version.